COMMUNICATIONS.md

```diff
@@ -0,0 +1,94 @@
+# OpenSearch Project Communication 
+
+- [Overview](#overview)
+- [Public Slack](#public-slack)
+  - [Getting Started](#getting-started)
+  - [Workspace Channels](#workspace-channels)
+  - [Tips](#tips)
+
+## Overview
+
+The purpose of this document is to provide references and information regarding
+the communication channels for the OpenSearch Project. All communication is
+subject to the [OpenSearch Code of Conduct](CODE_OF_CONDUCT.md). Please see
+[CONTRIBUTING](CONTRIBUTING.md) if you're interested in contributing to the
+project.
+
+## Public Slack
+
+OpenSearch has a workspace on [Slack](https://opensearch.slack.com) to provide
+public communication channels for all people interested in OpenSearch.
+
+The following guidelines include a Getting Started for steps to register and
+setup the workspace along with tips for using the workspace. Please read through
+these guidelines carefully and thoroughly. All communications and behavior on
+the public slack workspace is subject to the 
+[OpenSearch Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Getting Started
+
+_Join Us_
+
+  * Register your account at https://opensearch.slack.com. You will be required
+    to enroll in two-factor authentication to ensure the best possible security
+    of your account.
+
+_Update Profile_
+
+  * Add your interests or keywords in the “What I do” section of your profile 
+    along with your Title and Company/Organization. This is how participants 
+    can network and connect with each other by common interests. No anonymous 
+    display names are allowed.
```

Comment:
How do you plan to enforce this? 
Why? 
There may be good reasons for this (and we should be explicit as to what those are), but there are good reasons for not adopting this type of policy.
For context: https://en.wikipedia.org/wiki/Nymwars

Reply:
This is also a great point. I think the only slack enforcement is to disallow the shorter display names to be used in place of Full Name for a user's name display. The intent here (like above) was to make trolling more challenging. Since this is solving a problem that doesn't yet exist I'm also :100: okay with removing this and allowing shorter display names until it becomes an issue. Also curious what others think.

Reply:
+1. Let folk be folk. Deal with people being obtuse when it happens.